examples/with-biconomy-aa/src/viem.ts

```diff
@@ -0,0 +1,176 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+import prompts, { PromptType } from "prompts";
+import {
+  createWalletClient,
+  createPublicClient,
+  http,
+  type Account,
+  WalletClient,
+  formatEther,
+} from "viem";
+import { sepolia } from "viem/chains";
+import {
+  createSmartAccountClient,
+  BiconomySmartAccountV2,
+  PaymasterMode,
+} from "@biconomy/account";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+import { createAccount } from "@turnkey/viem";
+import { Turnkey as TurnkeyServerSDK } from "@turnkey/sdk-server";
+import { createNewWallet } from "./createNewWallet";
+import { print } from "./util";
+
+async function main() {
+  if (!process.env.SIGN_WITH) {
+    // If you don't specify a `SIGN_WITH`, we'll create a new wallet for you via calling the Turnkey API.
+    await createNewWallet();
+    return;
+  }
+
+  const turnkeyClient = new TurnkeyServerSDK({
+    apiBaseUrl: process.env.BASE_URL!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    defaultOrganizationId: process.env.ORGANIZATION_ID!,
+  });
+
+  // Initialize a Turnkey-powered Viem Account
+  const turnkeyAccount = await createAccount({
+    client: turnkeyClient.apiClient(),
+    organizationId: process.env.ORGANIZATION_ID!,
+    signWith: process.env.SIGN_WITH!,
+  });
+
+  const network = "sepolia";
+
+  // Bring your own provider (such as Alchemy or Infura: https://docs.ethers.org/v6/api/providers/)
```

Comment:
Well we kinda force them to use infura from our env, so perhaps we should just allow them to pass in a provider URL? But this is consistent with other examples - just noting. No action needed

Reply:
true! I'll reduce this comment for now